### ISSUE TITLE
Virtual Machine detection could be improved for VirtualBox

### ISSUE THREAD

**Issue by mckaygerhard** [lpereira/hardinfo#154]
i have several virtual box releases and virtual machine software/hardware for testing.. and some does not seem are detecting..

here a screen shot of virtualbox 3.2.4 russian that dont are detected in last git build, and the hardinfo report attached also:

![hardinfo-virtualboxdetection](https://user-images.githubusercontent.com/1571724/29318550-80686e9a-819e-11e7-89cc-8f9649883667.png)

[hardinfo_report.txt](https://github.com/lpereira/hardinfo/files/1225193/hardinfo_report.txt)

**Comment by mckaygerhard** [comment]
attached corrected report, but still the previous its valid, the previous report was 2 months ago.. the attached here are today: bot detect the machine type wrong:

[hardinfo_report.txt](https://github.com/lpereira/hardinfo/files/1225243/hardinfo_report.txt)

**Comment by bp0** [comment]
Can you try the tests from computer_get_virtualization() and see which test should succeed but doesn't?
https://github.com/lpereira/hardinfo/blob/master/modules/computer.c#L391

**Comment by mckaygerhard** [comment]
well :

1. for virtual machines get info from /proc/cpuinfo does not make sense
2. seems in debian 8+ doe snot exits anymore /proc/scsi path but in my older etch there are !
3. quite strange, /var/log/dmesg has 0 bytes!!!!! 

i suggest let as is those files and added two more.. :

1. /proc/bus/input/devices: if at least one string are "irtual" (without Box) maybe could be a virtual machine, but if those are "mouse" device, its for sure are virtual machines.
2. if dmesg are not available can also read /var/log/messages as alternative..
3. lspci output! iknow its a stream and not a file, but if at least one string has "Inotek"/"ware" and also "irtu" its a virtual machine for sure!

**Comment by bp0** [comment]
Does #157 solve this for you?

**Comment by mckaygerhard** [comment]
sorry men, errr that pull only makes translatable the strings and added extra checks to model name.., the :
```
    tmp = module_call_method("devices::getMotherboard");
    if (strstr(tmp, "VirtualBox") != NULL) {
```
its the only related change and does not make any sense..  in fact i tested then and still does not work..

**Comment by mckaygerhard** [comment]
i think all the ways to detect the machines must be based on varios string catures and not only in one/fisr provided/found

so the i must test then in the vmware sock but still not at the depot.... so if the pull will be merged still i need to test in the vmware sock to see if are good, due the patch are only to virtual box

**Comment by mckaygerhard** [comment]
I cannot test into the vmware slot for now.. ehantime make the issue related to virtualbox and closeit Fixed with b49c70650ec0a8bab5683d823dd46f46a42b8fb1
